src/components/BackEndRolePage/Brag.js

```diff
@@ -2,9 +2,18 @@ import React from 'react';
 
 import { CONFIDENT } from '../../constants/characterMoods';
 import { Robot } from '../Characters';
+import QuoteBubble from '../QuoteBubble';
+
+// TODO Replace with auto-generated message
+const QUOTE = 'Yesterday I was implementing PUT method for missing RESTful API endpoint.';
```

Comment:
I suggest to leave `QUOTE` on the higher level, since it is going to be replaced with some quoting service. We won't add import in every `Brag`, `Confess`, `Default` component, so we should progress closer to the solution with `props` passing.

Reply:
Honestly I would leave it as it is for now since it is the easiest to remove.

Reply:
Lol, it's not :D. It was easier to remove before your change :D